kubernetes/windows/setup.ps1

```diff
@@ -70,13 +70,19 @@ Write-Host ('Finished Extracting Certificate Generator Package')
 $windowsazuremonitoragent = [System.Environment]::GetEnvironmentVariable('WINDOWS_AMA_URL')
 if ([string]::IsNullOrEmpty($windowsazuremonitoragent)) {
     Write-Host ('Environment variable WINDOWS_AMA_URL is not set. Using default value')
-    $windowsazuremonitoragent = "https://github.com/microsoft/Docker-Provider/releases/download/windows-ama-bits/genevamonitoringagent.46.3.2.zip"
+    $windowsazuremonitoragent = "https://github.com/microsoft/Docker-Provider/releases/download/windows-ama-bits/GenevaMonitoringAgent.46.9.43.zip"
```

Comment:
are we picking new version of the AMA? if yes, then we would need to validate geneva path in addition to 3p path.

Reply:
This version had all the different bugs we asked AMA team to fix...Especially the bug where they wanted tagname to be exactly equal to the BLOB name. So, we will need the new version. I will add Geneva path testing to the testing scenarios